app/components/templates/CreateTemplateModal/SortablePageItem.tsx

```diff
@@ -0,0 +1,51 @@
+import React, { FunctionComponent, useCallback } from 'react';
+import i18n from 'i18next';
+
+import { useSortable } from '@dnd-kit/sortable';
+import { CSS } from '@dnd-kit/utilities';
+
+import { Icon, List, Ref } from '../../basic';
+
+interface SortablePageItemProps {
+    name: string;
+    onRemove: (value: string) => void;
```

Comment:
Minor, general:

This technically allows one item to tell that another item was removed :slightly_smiling_face: Probably not what we want. For more correctness, I would go for

```suggestion
    onRemove: () => void;
```

and the consumer using partial application/higher order functions to supply the `name`  to the callback

```tsx
name={name}
onRemove={() => onRemove(name)}
```